packages/perseus-editor/src/components/interactive-graph-settings.tsx

```diff
@@ -547,10 +548,12 @@ class InteractiveGraphSettings extends React.Component<Props, State> {
                                 </LabeledRow>
                             </div>
                             <div className="perseus-widget-left-col">
-                                <PropCheckBox
+                                <Checkbox
                                     label="Show tooltips"
-                                    showTooltips={this.props.showTooltips}
-                                    onChange={this.change}
+                                    checked={this.props.showTooltips}
+                                    onChange={(value) => {
+                                        this.change({showTooltips: value});
+                                    }}
```

Comment:
Suggestion: It seems like this might not have had coverage before, but it might be easy to cover with something like the following added to the `graph-settings.test.tsx` file. Looks like there are a number of places marked as missing coverage. Probably not necessary to fix all the coverage, but here's a possible test here at least:

```
test("calls onChange when tooltips label is changed", async () => {
        // Arrange
        const onChange = jest.fn();
        render(
            <GraphSettings
                editableSettings={["graph"]}
                showTooltips={true}
                onChange={onChange}
            />,
        );

        // Act
        const checkbox = screen.getByRole("checkbox", {
            name: "Show tooltips",
        });
        await userEvent.click(checkbox);

        // Assert
        expect(onChange).toHaveBeenCalledWith(
            expect.objectContaining({showTooltips: false}),
            undefined,
        );
    });
```